crates/wasmtime/src/compile.rs

```diff
@@ -66,6 +69,7 @@ pub(crate) fn build_artifacts<T: FinishedObject>(
     let mut translation = ModuleEnvironment::new(tunables, &mut validator, &mut types)
         .translate(parser, wasm)
         .context("failed to parse WebAssembly module")?;
+    translation.debuginfo.dwarf_package = dwarf_package;
```

Comment:
@alexcrichton  With the changes for the builder, are you anticipating this lines stays or goes?

Reply:
I'm going to assume it stays as does the extra parameter  to `build_artifacts` except that it will be `Option<gimli::DwarfPackage<EndianSlice<'_, LittleEndian>>>`

Reply:
I think this line is good to stay. This is sort of the choke point for "everyone before has to figure out how to thread it here" and "everyone after has to figure out how to read it from here".

The `build_artifacts` function is internal to Wasmtime so we can do whatever we want to its signature.